Enigmatry.Entry.AzureSearch.Tests/Searching/SearchServiceFixtureBase.cs

```diff
@@ -25,6 +25,9 @@ public async Task OneTimeSetup()
         WaitIndexToBeUpdated();
     }
 
+    [OneTimeTearDown]
+    public void OneTimeTearDown() => _services.Dispose();
```

Comment:
Why is this needed now. and does it conflict with the [TearDown] in a particular test? Will you dispose twice?

Reply:
Warning was reported by NUnit.Analyzers that it wasn't disposed. _services is initialized once in OneTimeSetup, and now disposed in OneTimeTearDown.